modules/kmm-customizing-upgrades-for-kernel-modules.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/kmm-kernel-module-management.adoc
+
+:_content-type: PROCEDURE
+[id="kmm-customizing-upgrades-for-kernel-modules_{context}"]
+= Customizing upgrades for kernel modules
+
+Use this procedure to upgrade the kernel module while running maintenance operations on the node,
```

Comment:
The sentence "Use this procedure to upgrade the kernel module while running maintenance operations on the node, including rebooting the node, if needed." is not very clear. 
```suggestion
Use this procedure to upgrade the kernel module while running maintenance operations on the node, including rebooting the node.
```

Reply:
s/including/ such as